python/pyspark/ml/recommendation.py

```diff
@@ -223,15 +220,14 @@ def setNumBlocks(self, value):
         Sets both :py:attr:`numUserBlocks` and :py:attr:`numItemBlocks` to the specific value.
         """
         self._set(numUserBlocks=value)
-        self._set(numItemBlocks=value)
+        return self._set(numItemBlocks=value)
```

Comment:
I guess we could do `return self._set(numUserBlocks=value)._set(numItemBlocks=value)`. But I think it's ok to leave this one.